### PR TITLE
refactor(styled-components): migrate NewIssue

### PR DESCRIPTION
**Pull request by samuelmeuli**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | – |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #532        |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after|

## Description

Migrated the `NewIssue` component to styled-components.

### CODE REVIEW

File: src/issue/screens/new-issue.screen.js [hunk 1]
```diff
@@ -126,16 +136,13 @@ class NewIssue extends Component {
               }
               onChangeText={text => this.setState({ issueTitle: text })}
               placeholderTextColor={colors.grey}
-              style={[
-                styles.textInput,
-                { height: Math.max(60, this.state.issueTitleHeight) },
-              ]}
+              style={[{ height: Math.max(60, issueTitleHeight) }]}
```
Comment (chinesedfan) [inline src/issue/screens/new-issue.screen.js]: Can you integrate `height` into styled-component?

File: src/issue/screens/new-issue.screen.js [hunk 2]
```diff
@@ -146,24 +153,20 @@ class NewIssue extends Component {
                 })
               }
               placeholderTextColor={colors.grey}
-              style={[
-                styles.textInput,
-                { height: Math.max(60, this.state.issueCommentHeight) },
-              ]}
+              style={[{ height: Math.max(60, issueCommentHeight) }]}
```
Comment (chinesedfan) [inline src/issue/screens/new-issue.screen.js]: And here.

**Comment by samuelmeuli** [comment]
Hi @chinesedfan, thanks for the feedback. I've updated the component.

Screenshots with text:

| Before   | After    |
| -------- | -------- |
|simulator screen shot - iphone 6 - 2018-10-23 at 10 47 12|simulator screen shot - iphone 6 - 2018-10-23 at 10 47 42|

**Comment by lex111** [comment]
@samuelmeuli thank you for contributing! Happy Hacktoberfest! :jack_o_lantern:

**Pull request merged**